jsk_mbzirc_common/gazebo_model/world/mbzirc_arena.world

```diff
@@ -32,10 +32,16 @@
     </include>
 
     <include>
-      <pose>0 0 0 1.57 -0 0</pose>
+      <pose>0 0 0 1.5708 -0 1.5708</pose>
```

Comment:
Because in my gazebo, I have to change the yaw to get the correct map
direction...


2016-01-13 22:33 GMT+08:00 Kei Okada <notifications@github.com>:

> In jsk_mbzirc_common/gazebo_model/world/mbzirc_arena.world
> <https://github.com/tongtybj/mbzirc/pull/21#discussion_r49596385>:
>
> > @@ -32,10 +32,16 @@
> >      </include>
> >
> >      <include>
> > -      <pose>0 0 0 1.57 -0 0</pose>
> > +      <pose>0 0 0 1.5708 -0 1.5708</pose>
>
> why you changed direction, i reverted this at 0bddf7f
> <https://github.com/tongtybj/mbzirc/commit/0bddf7f42e9603097c19e5646757b904eb910dec>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tongtybj/mbzirc/pull/21/files#r49596385>.
>